Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -396,7 +396,11 @@ def uninstall_rmdir(*directories, command: nil, **_)
               command.run!("/bin/rm", args: ["-f", "--", ds_store], sudo: true, print_stderr: false)
             end
 
-            command.run("/bin/rmdir", args: ["--", resolved_path], sudo: true, print_stderr: false)
+            paths = Dir["#{resolved_path}/*"]
```

Comment:
This isn’t recursive all the way down. That would be `Dir["#{resolved_path}/**/*"]`.

Reply:
@vitorgalvao It is fully recursive, as the contents of the current directory are passed again to `uninstall_rmdir` each time, as long as they consist of only empty directories… I can confirm it working for any depth of empty subfolders. Try it yourself… Also `**` can be slow!

There _may_ be a more efficient way of implementing this overall that I'm missing. Feel free to chime in with some code if you have a better approach… But this is 100% working as it stands.